src/imgtools/utils/interlacer_utils.py

```
@@ -0,0 +1,309 @@
+from dataclasses import dataclass, field
+from rich.console import Console
+from rich.highlighter import RegexHighlighter
+from rich.theme import Theme
+from rich.tree import Tree as RichTree
+from typing import TYPE_CHECKING
+
+from imgtools.loggers import logger
+from imgtools.utils import OptionalImportError, optional_import
+
+if TYPE_CHECKING:
+    from rich.repr import RichReprResult
```

Ruff (0.11.9)

`rich.repr.RichReprResult` imported but unused

Remove unused import: `rich.repr.RichReprResult`

(F401)